src/basics.md

```diff
@@ -1101,6 +1101,68 @@ fn main() {
 }
 ```
 
+[ex-error-chain-backtrace]: #ex-error-chain-backtrace
+<a name="ex-error-chain-backtrace"></a>
+## Obtain backtrace of complex error scenarios
+
+[![error-chain-badge]][error-chain] [![cat-rust-patterns-badge]][cat-rust-patterns]
+
+Extend errors by appending new errors via [`chain_err`]. It provides a
+better context to understand an error in a form of a backtrace. The
+below recipes attemps to deserialize the value `256` into a `u8`. An
```

Comment:
Also I would present the example backtrace output after the code snippet.

Reply:
Isn't it redundant if the reader hit the play button?

Reply:
We will be only able to (easily) simulate one error scenario (file not found). We might decide to show backtraces from other error scenarios too. But we will see how the example unfolds :)